### ISSUE TITLE
ubuntu bionic arm64：standard_init_linux.go:211: exec user process caused "no such file or directory"

### ISSUE THREAD

**Issue by Xphobia** [clear-code/td-agent-builder#90]
# Problem
rake apt:build APT_TARGETS="ubuntu-bionic-arm64" failed
my host is ubuntu bionic  on x86_64 platform

# How to reproduce

Follow the instructions in Readme to build
```
export TARGET="ubuntu-bionic-arm64"
sudo apt install qemu-user-static
cd /path/to/td-agent-builder
cp /usr/bin/qemu-aarch64-static td-agent/apt/${TARGET}
rake apt:build APT_TARGETS=${TARGET}
```

# Expected
build successfully

# Actual
build failed

# More infomation

![$%H6{MRDJN`N43C8KBGR248](https://user-images.githubusercontent.com/59901623/84121427-bdc03600-aa69-11ea-89ac-53a3fe02bbbb.jpg)

**Comment by ashie** [comment]
Thanks for the report. It's reproduced on my environment.
On the other hand, when I copy it to ubuntu-bionic instead of ubuntu-bionic-arm64, it works.
The instruction seems wrong, or changes of something may affect to it.
I'll check it.

**Comment by Xphobia** [comment]
> Thanks for the report. It's reproduced on my environment.
> On the other hand, when I copy it to ubuntu-bionic instead of ubuntu-bionic-arm64, it works.
> The instruction seems wrong, or changes of something may affect to it.
> I'll check it.

**Comment by Xphobia** [comment]
sorry, i just mis-clicked somehow, which leads to close this issue. i've reopened it now . BTW，i search about this error by google . it says the cause is Dockerfile whith dos format. [ 
How to fix “exec user process caused „no such file or directory“” in Docker](https://futurestud.io/tutorials/how-to-fix-exec-user-process-caused-no-such-file-or-directory-in-docker)

**Comment by Xphobia** [comment]
i find a word error in README. the upper one should be apt .
![M2YG%NH_X)P~F DO5I{Q@5X](https://user-images.githubusercontent.com/59901623/84219862-04626e80-ab04-11ea-963f-6cdd32f48d48.png)

**Comment by ashie** [comment]
> i find a word error in README. the upper one should be apt .

Thanks, I've fixed it.

**Comment by ashie** [comment]
> BTW，i search about this error by google . it says the cause is Dockerfile whith dos format.

It's not our case. The cause of our problem is missing qemu-aarch64-static. I'll fix README.md.

**Comment by ashie** [comment]
I've fixed the instruction via #91

**Comment by Xphobia** [comment]
> I've fixed the instruction via #91

got it. thanks a lot.
